lib/SILGen/SILGen.cpp

```diff
@@ -967,13 +967,20 @@ void SILGenModule::emitFunctionDefinition(SILDeclRef constant, SILFunction *f) {
     auto *init = constant.getInitializationExpr();
     assert(init);
 
+    auto *parentMod = f->getParentModule();
```

Comment:
Is `parentMod` expected to be non-null? If so, is it useful to assert that? Or it. could be null?

Reply:
It's generally going to be nonnull, but it would be ok if it was null as well, since we just don't want to erase what's there already, so i don't think it's worth the assertion here.

Reply:
thank you for the clarification